src/MovieCard/MovieCard.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 import './MovieCard.css';
+import tomato from '../images/tomato.svg';
+import badTomato from '../images/tomato-bad.svg';
 
 const MovieCard = ({ image, title, rating, id, movieClicked }) => {
 
+  let icon = null;
+
+  if (rating >= 5) {
```

Comment:
Could we use a ternary here instead? Also, does this need to wrapped in method instead?